nvflare/app_common/aggregators/collect_and_assemble_aggregator.py

```diff
@@ -46,13 +46,17 @@ def accept(self, shareable: Shareable, fl_ctx: FLContext) -> bool:
             self.log_error(fl_ctx, "no data to aggregate")
             return False
         data = dxo.data
+        meta = dxo.meta
+
         current_round = fl_ctx.get_prop(AppConstants.CURRENT_ROUND)
-        return self._accept_contribution(contributor_name, current_round, data, fl_ctx)
+        return self._accept_contribution(contributor_name, current_round, data, meta, fl_ctx)
```

Comment:
Why not use DXO here instead of data + meta?

Reply:
good idea! Updated to use DXO directly